Alignment/OfflineValidation/macros/makeArrowPlots.C

```diff
@@ -74,10 +77,14 @@ int makeRPhiArrowPlot( TTree* data, const char* name, double xLim, double yLim,
     }
   DrawRPhiLegend( xLim, yLim, barrelRPhiRescale );
 
-  char sliceLeg[192];
-  sprintf( sliceLeg, "%s: %f < z <= %f", name, zMin, zMax );
+  //char sliceLeg[192];
+  std::ostringstream sliceLeg_s;
+  sliceLeg_s << name << ": " << zMin << " < z <= " << zMax;
+  TString sliceLeg(sliceLeg_s.str());
+  //sprintf( sliceLeg, "%s: %f < z <= %f", name, zMin, zMax );
   //Plot10Mu( name, xLim/2, yLim, 0.2*xLim );
   TPaveText* atext = new TPaveText(0.2*xLim,0.85*yLim,0.66*xLim,0.99*yLim);
+  //atext->AddText(sliceLeg);
```

Comment:
you added a commented line... either explain why this might be needed or remove

Reply:
Cleaned.